File: javascript/starPattern.js
function printStarPattern(input) {
	if (input > 0) {
		for( let i = 1; i <= input; i++) {
			let str = "";
			for (let j = 1; j <= i; j++) {
				str += "*"
			}
			console.log(str);
		}
	} else {
		console.log("Input should be +ve");
	}
}

printStarPattern(5);

File: c/starPattern.c
#include <stdio.h>

void printStarPattern(int input) {
	if (input > 0) {
		for (int i = 1; i <= input; i++) {
			for (int j = 1; j <= i; j++) {
				printf("*");
			}
			printf("\n");
		}
	} else {
		printf("Input should be +ve\n");
	}
}

int main() {
	printStarPattern(6);
}